extensions/database/src/com/google/refine/extension/database/DBQueryResultImportReader.java

```diff
@@ -208,14 +208,12 @@ private List<List<Object>> getRowsOfCells(int startRow) throws IOException, Data
                 }
 
                 rowsOfCells.add(rowOfCells);
-
             }
 
         }
+
         end = dbRows.size() < batchSize + 1;
```

Comment:
This looks suspicious. Should the `+ 1` be removed to align with the other change you made?